test/images/metadata-concealment/check_metadata_concealment.go

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Why do we need this additional file?
I could not get the point from the pr message.